pkg/batchd/policy/drf/podset_info.go

```diff
@@ -30,6 +30,7 @@ type podSetInfo struct {
 	priority         float64
 	total            *cache.Resource
 	pendingIndex     int
+	assignedPending  int
```

Comment:
is `pendingIndex` still necessary?

Reply:
yes, `pendingIndex` is used to skip the pending pods(with NodeName) quickly

Reply:
great; please also try build an iterator for pod loops in other PR.